### DISCUSSION TITLE
User interface wording-related ideas

### DISCUSSION THREAD

**Discussion by revcor** [Anze/KeyCluCask#37] [Ideas]
A few suggestions I thought of regarding some interface text

### For app preferences window

**General pane**

_Launch at Login_ option

- In the “Not supported” note below the setting name, change  “…to the Login Items…”  to  “…to Login Items…”
- (Remove “the” because Login Items is the name of a specific section in System Preferences, not a generic description)

**Visual pane**

_Show App icon_ option

- In the option name, replace “Show App icon” with “Show app icon”
- (The word “app” should begin with lowercase letter, both to adhere to rules of written English and also to keep consistency with how the other settings’ names are written)
- (Is a setting to control such a minor, inconsequential visual detail necessary?)

All three options under _Menu Items_

- Add description text below each option, or tool tips with descriptions
- (I have never heard the terms used before, but if “bookmarked menu items” or “hidden/inactive elements” are common knowledge for the average user, and I am unique in my ignorance, then ignore this suggestion)

_Truncate long elements_ setting

- In the description text below, two possible suggestions for more natural sounding English phrasing:
- “Certain menu items with long names may, depending on column layout, be split over multiple lines”
- “When enabled, truncates longer menu items to avoid overflowing onto multiple lines”
- (I think those are both improvements, but they were just the first things that came to mind, and can likely be improved upon further)

**Advanced pane**

_Show Localized Custom Shortcuts_ option

- In the option name, change  “Show Localized Custom Shortcuts”  to  “Show localized custom shortcuts”
- (After capitalized first word, following words should begin with lowercase letters for consistency)

### General suggestions

- Consider replacing the word “element,” throughout the app, with a different term to refer to the individual shortcuts listed by the app
- I believe that is a common and correct term on the developer side, but less familiar to the average person/end user. It may not immediately be clear to every user what “elements” are
- Perhaps a term like “shortcut items” or “list items” or “keyboard shortcuts” to make it very plain and clear what is being referred to, regardless of computer expertise or experience
- Ideally someone’s 90-year-old grandmother would be able to look through the app preferences and understand the function of each setting

**Comment by Anze** [comment]
Thank you for your suggestions. I've updated the app with improved wording, so it should now appear and read more clearly.
